en/user_guide/content.rst

```diff
@@ -0,0 +1,66 @@
+Contents
+========
+
+Contens are instances of a Content Types. A Content Type is the pattern of a family of Contents. If
+"Car" was a Content Type, "Ford mustang" or "Dodge Viper" could be contents of that type.
+See `Content Types`_ documentation for more information.
+
+.. _Content Types: ./content_types.rst
```

Comment:
We should be consistent over the doc and always put the links at the end of the files.